compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/loops/HeaderProcessor.kt

```diff
@@ -336,7 +336,7 @@ internal class HeaderProcessor(
     }
 
     private fun DeclarationIrBuilder.ensureNotNullable(expression: IrExpression) =
-        if (expression.type is IrSimpleType && expression.type.isNullable()) {
+        if (expression.type is IrSimpleType && expression.type.isNullable) {
```

Comment:
In the future, please extract refactorings that don't change behavior (function -> property conversion, renames, moves between packages) into separate commits. This simplifies review a lot.

Reply:
Will do! To be honest I only refactored the code since the PR was touching every file which used this utility function anyway.